CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## 0.0.1-alpha.14
+
+### Bug fixes
+
+- Fix descriptor lookups for most types that overlap with `None` ([#19120](https://github.com/astral-sh/ruff/pull/19120)).
+    This means that e.g. `object().__str__()` now correctly binds the `self` argument of the `__str__`
+    method, as the `object` type overlaps with `None`.
+- Don't allow first-party code to shadow the stdlib `types` module ([#19128](https://github.com/astral-sh/ruff/pull/19128)).
+    This fixes a possible source of stack overflows.
+- Add cycle detection to our ty's implementation of disjointness, preventing another possible source of stack overflows when analysing recursive types ([#19139](https://github.com/astral-sh/ruff/pull/19139))
+
+### Server
+
+- Filter a symbol from a stub file in autocomplete suggestions if it is an implementation detail of the stub ([#19121](https://github.com/astral-sh/ruff/pull/19121))
+
```

Comment:
Minor: Can we remove these empty lines in the list here? They have an effect on the formatting. If you did this intentionally for some reason, feel free to keep it of course.